include/inviwo/core/util/zip.h

```diff
@@ -469,12 +469,12 @@ struct sequence {
         }
 
         iterator& operator+=(difference_type rhs) {
-            val_ += rhs * inc_;
+            val_ += static_cast<value_type>(rhs) * inc_;
```

Comment:
This is a general problem with unsigned types. Using a negative increment also breaks the sequence for uint32_t.

Reply:
But it would have worked before you added the cast... I might give a warning, but the result would be right, now the result is just wrong and with no warning...

Reply:
That is true. Will remove the cast. Any suggestions whether we should handle negative increments for unsigned types and how?

Reply:
we don't need to handle negative values for inc_ that will give a warning in the constructor of sequence. Only sequence += difference_type(-1); 
Have not figured out the proper set of cast to do that correctly yet...